test/Microsoft.ML.Tests/Scenarios/PipelineApi/PipelineApiScenarioTests.cs

```diff
@@ -47,15 +47,15 @@ public class IrisPrediction
         public class SentimentData
         {
             [Column("0", name: "Label")]
-            public bool Sentiment;
+            public Runtime.Data.DvBool Sentiment;
```

Comment:
This seems to be going backwards.  Why do we need this?

Reply:
Reverted.

---
In reply to: [212431987](https://github.com/dotnet/machinelearning/pull/683#discussion_r212431987) [](ancestors = 212431987)